readme.md

```diff
@@ -138,6 +146,24 @@ Returns a [`child_process` instance](https://nodejs.org/api/child_process.html#c
   - is also a `Promise` resolving or rejecting with a [`childProcessResult`](#childProcessResult).
   - exposes the following additional methods and properties.
 
+#### kill([signal], [options])
+
+Same as the original [`child_process.kill()`](https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal) except: if `signal` is `SIGTERM` (the default value) and the child process is not terminated after 5 seconds, force it by sending `SIGKILL`.
+
+##### options.retry
+
+Type: `boolean`<br>
+Default: `true`
+
+If the first signal does not terminate the child after a specified timout, a `SIGKILL` will be sent to the process.
+
+##### options.retryAfter
```

Comment:
I think `retryAfter` => `forceKillAfter` would be clearer.

Reply:
We should rename `retry` to `forceKill` as well. It actually is not really a "retry" since we are somewhat guaranteed to succeed.

Reply:
Done: 57a54749eda4a6cc4013f31dda7adcc34dcd1f80